extensions/wikia/UserProfilePageV3/UserProfilePageController.class.php

```diff
@@ -724,16 +436,11 @@ private function renderAvatarLightbox( $userId ) {
 		$user = User::newFromId( $userId );
 
 		$this->setVal( 'defaultAvatars', $this->getDefaultAvatars() );
-		$this->setVal( 'isUploadsPossible',
-			$this->wg->EnableUploads &&
-			$this->wg->User->isAllowed( 'upload' ) &&
-			empty( $this->wg->AvatarsMaintenance )
-		);
 
 		$this->setVal( 'avatarName', $user->getGlobalAttribute( AVATAR_USER_OPTION_NAME ) );
 		$this->setVal( 'userId', $userId );
 		$this->setVal( 'avatarMaxSize', self::AVATAR_MAX_SIZE );
```

Comment:
I don't think this is being used anymore, right? AFAICT the only usage was [here](https://github.com/Wikia/app/pull/16081/files?utf8=%E2%9C%93&diff=split#diff-fa16031792b15f1939e781da1d596a33L30).

Reply:
Yeah :) Removed in 5fca599